meta-ostro/recipes-image/images/ostro-image.bb

```diff
@@ -182,7 +182,7 @@ COMPRESS_DEPENDS_zip = "zip-native"
 
 # Replace the default "live" (aka HDDIMG) images with whole-disk images
 # XXX Drop the VM hack after taking care also of the non UEFI devices (those using U-Boot: edison and beaglebone)
-OSTRO_VM_IMAGE_TYPES ?= "dsk dsk.xz dsk.vdi"
+OSTRO_VM_IMAGE_TYPES ?= "dsk dsk.sha256sum dsk.bmap dsk.xz dsk.vdi dsk.vdi.sha256sum dsk.vdi.bmap"
```

Comment:
potentially, you can use bmaptool to efficiently copy files from remote location with preserving sparseness on filesystem.